indico/modules/events/client/js/util/social.js

```diff
@@ -55,8 +42,6 @@
       },
       events: {
         render() {
-          injectFacebook($('.social-button-container').data('social-settings').facebook_app_id);
-          $.getScript('//apis.google.com/js/plusone.js');
           $.getScript('//platform.twitter.com/widgets.js');
```

Comment:
Is this still needed?

Reply:
Now has been replaced with a 'spyware free' button